src/components/NewtorkAlert.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import tw from "twin.macro";
+import Portal from "./Portal";
+import { useConnection } from "../hooks";
+import { SUPPORTED_NETWORK_IDS } from "../config";
+
+const Alert: React.FC = () => {
+  const { isConnected, network } = useConnection();
+
+  const isNetworkUnsupported =
+    isConnected &&
+    network &&
+    !SUPPORTED_NETWORK_IDS.includes(network.chainId as 1 | 42);
```

Comment:
this is a misleading cast, though i dont think it will break anything. but we want to show we are checking any chainId, not just 1 or 42

Reply:
Good point, changed